route.rb

```diff
@@ -1 +1,19 @@
+class Route
+  attr_reader :stations
 
+  def initialize(start, finish)
+    @stations = [start, finish]
+  end
+
+  def add_station(station)
+    @stations.insert(-2, station)
+  end
+
+  def delete_station(station)
+    @stations.delete_at(station) if @stations.first != station && @stations.last != station
```

Comment:
а еще я рекомендовал написать методы (геттеры) для первой и последней станций
с их использованием можно вот так записать:
```suggestion
    @stations.delete_at(station) unless [start, finish].include? station
```
(можно и без них через "инклюд", но с ними чище)